code/modules/reagents/chems/chems_ethanol.dm

```diff
@@ -9,23 +9,27 @@
 	solvent_power = MAT_SOLVENT_MODERATE
 	uid = "chem_ethanol"
 
-	heating_message = "boils away its water content, leaving pure ethanol."
-	heating_point = T100C + 10
+	boiling_point = T0C + 78.37
+
+	heating_message = "boils away its ethanol content, leaving pure water."
+	heating_point = T0C + 78.37
 	heating_products = list(
 		/decl/material/liquid/ethanol = 0.75,
 		/decl/material/liquid/water =   0.25
 	)
 	bypass_heating_products_for_root_type = /decl/material/liquid/ethanol
 
 	chilling_message = "separates as its water content freezes, leaving pure ethanol."
-	chilling_point = T0C - 10
+	chilling_point = T0C
 	chilling_products = list(
 		/decl/material/liquid/ethanol = 0.75,
-		/decl/material/liquid/water =   0.25
+		/decl/material/solid/ice =      0.25
 	)
 	bypass_chilling_products_for_root_type = /decl/material/liquid/ethanol
 	affect_blood_on_ingest = FALSE // prevents automatic toxins/inebriation as though injected
 
+	temperature_burn_milestone_material = TRUE
```

Comment:
this is not unset on subtypes of ethanol, i think the only reason it doesn't explode is because they don't have a boiling point set. if that ever changes it will say "it is burning hot enough to boil [every single subtype of ethanol]" and that's... unpleasant

my suggestion is either set it to false in the Initialize override below, or go with the prior suggestion of making `temperature_burn_milestone_material` a typepath instead of a bool.